src/Controller/MainController.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/*
+ * This file is part of the JoliTypo Website project.
+ * (c) JoliCode <coucou@jolicode.com>
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace App\Controller;
+
+use App\Forms\FixerFormType;
+use App\Services\FixerDataHolder;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+class MainController extends AbstractController
+{
+    /**
+     * @Route("/", name="homepage")
```

Comment:
*`MainController` vs `homepage` (route name) vs `index` (method name). This is not very consistant